cola/inotify.py

```diff
@@ -199,6 +204,10 @@ def _add_watch_failed_warning(self, directory, e):
         core.stderr('    sudo tee -a /etc/sysctl.conf &&')
         core.stderr('    sudo sysctl -p\n')
 
+    def _is_ignored(self, path):
+        # Ignore all Git paths except "HEAD" (notify on branch switch or other type of checkout)
+        return (path.startswith('.git') or '/.git/' in path) and not path.endswith('HEAD')
```

Comment:
The `path.startswith('.git')` check, will that make it blind to .gitignore changes?  Instead of `endswith('HEAD')`, maybe `os.path.basename(path) == 'HEAD'` instead?